modules/consensus/accept_test.go

```diff
@@ -1145,7 +1148,22 @@ func TestChainedAcceptBlock(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if bcs.changes != 3 || bcs.appliedBlocks != int(cst2.cs.Height()+1) || bcs.revertedBlocks != 0 {
+	if bcs.appliedBlocks != int(cst2.cs.Height()+1) || bcs.revertedBlocks != 0 {
 		t.Error("consensus changes do not seem to be getting passed to subscribers correctly")
 	}
+
+	// Check that every change recorded in 'bcs' is also available in the
+	// consensus set.
+	for _, change := range bcs.changes {
+		err := cst2.cs.db.Update(func(tx *bolt.Tx) error {
+			_, exists := getEntry(tx, change)
```

Comment:
ah yeah, little lazy about the performance here. Maybe adds 1 second to the test, shouldn't be too bad.